src/ker_lowupsampfac_horner_allw_loop_constexpr.c

```diff
@@ -2,191 +2,170 @@
 // Authors: Alex Barnett & Ludvig af Klinteberg.
 // (C) The Simons Foundation, Inc.
   if constexpr(w==2) {
-    FLT c0[] = {2.3711015472112514E+01, 2.3711015472112514E+01, 0.0000000000000000E+00, 0.0000000000000000E+00};
-    FLT c1[] = {2.5079742199350562E+01, -2.5079742199350562E+01, 0.0000000000000000E+00, 0.0000000000000000E+00};
-    FLT c2[] = {-3.5023281580177050E+00, -3.5023281580177086E+00, 0.0000000000000000E+00, 0.0000000000000000E+00};
-    FLT c3[] = {-7.3894949249195587E+00, 7.3894949249195632E+00, 0.0000000000000000E+00, 0.0000000000000000E+00};
+    FLT c0[] = {6.1209111871385724E-01, 6.1209111871385691E-01, 0.0000000000000000E+00, 0.0000000000000000E+00};
```

Comment:
We do not need padding anymore. The CPU version generates it (even when xsimd is not used explicitly. The GPU version does not use padding.